gspread/models.py

```diff
@@ -744,14 +744,15 @@ def get_all_records(
         head=1,
         default_blank="",
         allow_underscores_in_numeric_literals=False,
+        numericise_ignore=[]
```

Comment:
A [mutable default argument](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) may lead to surprising behavior.

Reply:
You're quite right. Fixed.